### ISSUE TITLE
[BUG] Nest-CLI Auto-Importer is Importing Files in incorrect place

### ISSUE THREAD

**Issue by shekohex** [nestjs/nest-cli#191]
## I'm submitting a...
<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[ ] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
Generator Errors
when i tried to run 
```bash
$ nest g mo app/advertisement
```
the result of this command is OK!
```bash
CREATE /src/app/advertisement/advertisement.module.ts (90 bytes)
UPDATE /src/app/app.module.ts (1963 bytes)
```
**But** !
Before:
![image](https://user-images.githubusercontent.com/14620076/46629642-e3cd5e80-cb41-11e8-86a8-0ad414e021b0.png)

After:
![image](https://user-images.githubusercontent.com/14620076/46629537-7b7e7d00-cb41-11e8-8dfc-78693202660e.png)
 
## Expected behavior
To add imports in correct place.

## Minimal reproduction of the problem with instructions
create a new nest project
add some imports in the file `app.module.ts` - _the root module_ - in that way
```ts
import {
    ModuleA,
    ModuleB,
    ModuleC
} from './modules'
```
then try to generate new module
## What is the motivation / use case for changing the behavior?
I think the problem that `nest-cli` only matches the last import `import {`
and start adding the new import in the new line, but it should also matches the `}` at the end of that line *and* the **new lines**

## Environment

```
$ nest info                  

[System Information]
OS Version     : Linux 4.18
NodeJS Version : v10.10.0
NPM Version    : 6.4.1
[Nest Information]
websockets version : 5.3.11
mongoose version   : 5.2.2
swagger version    : 2.5.1
typeorm version    : 5.2.2
common version     : 5.3.11
core version       : 5.3.11
```

**Comment by kamilmysliwiec** [comment]
Hey @shekohex,
I'm trying to reproduce your issue. Do you still face this issue? (after latest updates)

**Comment by shekohex** [comment]
Hi Kamil,

unfortunately, Yes 

```bash
$ nest g mo app/user-transactions2                
CREATE /src/app/user-transactions2/user-transactions2.module.ts (94 bytes)
UPDATE /src/app/app.module.ts (2359 bytes)

```

![image](https://user-images.githubusercontent.com/14620076/48148819-5a858500-e2c3-11e8-9312-0fc7d78edca0.png)

`nestjs/cli v5.6.2`


```
$ nest info                       

[System Information]
OS Version     : Linux 4.18
NodeJS Version : v10.10.0
NPM Version    : 6.4.1
[Nest Information]
websockets version : 5.3.11
mongoose version   : 5.2.2
swagger version    : 2.5.1
typeorm version    : 5.2.2
common version     : 5.3.11
core version       : 5.3.11
```

**Comment by kamilmysliwiec** [comment]
Could you share a small repo so I can reproduce this issue locally?

**Comment by shekohex** [comment]
Here we go
https://github.com/shekohex/module-import-test

try to add new module

`$ nest g mo module8`

and see the main module

**Comment by kamilmysliwiec** [comment]
Thanks @shekohex. Looks like we have to update underlying `@angular-devkit/schematics` engine :) I'll do it soon

**Comment by kamilmysliwiec** [comment]
Fixed in 5.7.0. Thanks for reporting! 💪
